packages/babel-generator/test/fixtures/types/DoExpression/actual.js

```diff
@@ -0,0 +1,7 @@
+let a = do {
```

Comment:
Given your test case, when it fails, you might not be sure that it is actually the `DoExpression` that is failing. So in this case, I would argue that we might want to go for a more isolated test like:
```js
(do {})
```